### ISSUE TITLE
Looks like the missing production panel issue is back

### ISSUE THREAD

**Issue by MarkusKV** [Azurency/CQUI_Community-Edition#243]
A city failed to complete a wonder and now its production panel is missing.

Is there a keyboard shortcut to select something to build?
Is there a way to skip building something altogether?
I'm guessing that getting to the next turn might help.

**Comment by MarkusKV** [comment]
Getting to the next turn doesn't help.  I can continue to play, but one city isn't building anything.
I'll collect any information that might be useful.  Let me know.  A save file, perhaps?

**Comment by MarkusKV** [comment]
Yay.  Something I did got it working.  Sometimes, the city view shows no production panel, but advancing to the next city then back to your city will show the production panel and queue for the wrong city.  Selecting something to build seems to have no effect, but advancing to the next city I think I saw it added to the build queue.  Going back to the original city, then choosing 'reset' on the queue seems to have done it.  The production panel and queue are working.  

Anyway, something I did made it work.
